composer.json

```diff
@@ -7,12 +7,13 @@
     "require": {
         "php": "^7.1.3",
         "backup-manager/laravel": "^1.2",
-        "laracasts/presenter": "^0.2.1",
+        "doctrine/dbal": "^2.9",
```

Comment:
Move this `doctrine/dbal` dependency to `require-dev`, since is only used for sqlite, expecially just for running tests.